Code/Framework/AzCore/AzCore/Debug/Trace.h

```diff
@@ -25,6 +25,9 @@ namespace AZ
 {
     namespace Debug
     {
+        // The NoWindow constant contains a special case string that can skip outputting the
+        // "<window-name> :" portion of a trace message of the form "<window-name>: <message>"
+        inline constexpr const char* NoWindow = "<no-window>";
```

Comment:
Why does this specifically use the string "<no-window>" instead of an empty string? It seems like just passing in an empty string should also skip printing the window?

Reply:
To provide some background on the current behavior of passing in a `nullptr` vs passing in `""` is different.
Passing in a `nullptr` would fallback to using the default window of "System" as seen [here](https://github.com/o3de/o3de/blob/9f18b8fff64778585652a567b8265618962ae370/Code/Framework/AzCore/AzCore/Debug/Trace.cpp#L592-L595).

But passing in "" string would result in the window being skipped.

I will switch over the NoWindow constant to be `""` now, since the behavior of passing in the "<no-window>" or empty string acts the same now.